README.md

```diff
@@ -66,12 +66,7 @@ EOF
 
 ### Install net-gateway-api
 ```bash
-ko apply -f config/
-```
-
-### Load tested environment versions
-```
-source ./hack/test-env.sh
+kubectl apply -f https://github.com/knative-extensions/net-gateway-api/releases/latest/download/net-gateway-api.yaml
```

Comment:
The instructions are more for development instead of installing the released bits.

What would make sense is to update our knative.dev website with docs on how to perform this installation and leave the README.md as is

Reply:
Got it I will close this and progress there